src/accelerate/utils/modeling.py

```diff
@@ -840,10 +843,23 @@ def load_state_dict(checkpoint_file, device_map=None):
             # all weights that haven't defined a device should be loaded on CPU
             device_weights["cpu"].extend([k for k in weight_names if k not in sum(device_weights.values(), [])])
             tensors = {}
+            pbar = is_tqdm_available() and tqdm(
+                main_process_only=False,
+                total=sum([len(device_weights[device]) for device in devices]),
+                unit="w",
+                smoothing=0,
+                leave=False,
+            )
             for device in devices:
                 with safe_open(checkpoint_file, framework="pt", device=device) as f:
                     for key in device_weights[device]:
+                        if pbar:
```

Comment:
```suggestion
                        if progress_bar is not None:
```
then update pbar to progress_bar